src/layouts/Home/components/News.tsx

```diff
@@ -8,12 +8,18 @@
 import React from 'react';
 import { useStaticQuery, graphql } from 'gatsby';
 import styled, { css } from 'styled-components';
-import { getColor } from '@zendeskgarden/react-theming';
+import { SELECTOR_FOCUS_VISIBLE, getColor } from '@zendeskgarden/react-theming';
 import { Grid, Row, Col } from '@zendeskgarden/react-grid';
 import { XXL } from '@zendeskgarden/react-typography';
 import { Anchor } from '@zendeskgarden/react-buttons';
 import MaxWidthLayout from 'layouts/MaxWidth';
 
+const StyledNewsAnchor = styled(Anchor)`
+  ${SELECTOR_FOCUS_VISIBLE} {
+    color: ${p => getColor('primaryHue', 700, p.theme)};
```

Comment:
The `:active` color change seen on the production website is missing when the news links have focus.